src/events/eventKeyboard.lua

```diff
@@ -11,5 +11,7 @@ function eventKeyboard(name, key, down, x, y)
         displayProfile(Player.players[name], name)
     elseif key == keys.LETTER_L then
         leaderboard.displayLeaderboard("global", 1, name)
+    elseif key == keys.LETTER_O or key == keys.LETTER_U then
+	shop.displayShop(name, 1)
```

Comment:
The indentation doesn't match with the lines of the same level (8 spaces). There are some indentation problems in many places of the code anyway, so that won't be a big problem. But would be better if you just fix it :P